build-configs/common/theme/colors.ts

```diff
@@ -36,6 +38,8 @@ export const commonLightColors = {
   invalidInput: '#B3261E',
   warningColor: '#FFA726',
   linkColor: '#0b57d0',
-  grayBackgroundColor: '#dedede',
-  slightlyDarkGray: '#b9b9b9',
+  ttsPlayerWarningBackground: 'rgba(255, 253, 230, 1)',
```

Comment:
:wrench: is there any reason for using rgba here?
If not please convert it to hex